### PR TITLE
integration/kubernetes: wait for registry container

### LINKED ISSUE #3865: registry: read: connection reset by peer
Kubernetes test fails randomly on bare metal, looks like registry container is not ready when the `stress-kata` images is pushed

```
20:35:02 The push refers to repository [localhost:5000/stress-kata]
20:35:02 Get http://localhost:5000/v2/: read tcp 127.0.0.1:45042->127.0.0.1:5000: read: connection reset by peer
20:35:02 [init.sh:120] ERROR: sudo -E docker push localhost:5000/stress-kata:latest
20:35:02 [pmem_test.sh:42] ERROR: integration/pmem/../../integration/kubernetes/init.sh
20:35:03 The connection to the server localhost:8080 was refused - did you specify the right host or port?
20:35:03 The connection to the server localhost:8080 was refused - did you specify the right host or port?
20:35:03 integration/pmem/pmem_test.sh: line 50: pushd: pmem-csi: No such file or directory
20:35:03 error: the path "deploy/common/pmem-kata-app.yaml" does not exist
20:35:03 error: the path "deploy/common/pmem-kata-pvc.yaml" does not exist
20:35:03 error: the path "deploy/common/pmem-storageclass-ext4-kata.yaml" does not exist
```

cc @Jakob-Naucke  @GabyCT

### PR DESCRIPTION
**Pull request by devimc**
Wait for registry container before pushing stress-kata image

fixes #3865

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
cc @Jakob-Naucke @GabyCT

**Comment by GabyCT** [comment]
please put the label if it needs or not a backport

File: integration/kubernetes/init.sh [hunk 1]
```diff
@@ -117,6 +117,13 @@ sudo systemctl status "${cri_runtime}" --no-pager
 if [ "${stress_image_pull_policy}" == "Always" ]; then
 	echo "Store custom stress image in registry"
 	sudo -E "${container_engine}" run -d -p ${registry_port}:5000 --name "${registry_name}" "${registry_image}"
+	# wait for registry container
+	for i in $(seq 1 5); do
```
Comment (Jakob-Naucke) [inline integration/kubernetes/init.sh:121]: You could use our https://github.com/kata-containers/tests/blob/4cd975e0067a1c3ff59829d8d5c476c2f4e7e60d/.ci/lib.sh#L173 here

**Comment by devimc** [comment]
changes applied, thanks @GabyCT and @Jakob-Naucke

**Review by Jakob-Naucke — APPROVED** [review]

**Comment by Jakob-Naucke** [comment]
re ARM & s390x failures: Fedora's registry is down, I have nothing to do with this

**Pull request merged**